README.md

```diff
@@ -101,6 +103,23 @@ Remember you'll need to run `bundle install` and `yarn install` on an ongoing ba
      - If you run into problems with `solr_wrapper`, you can also start Solr with `rake figgy:development`
    - Access Figgy at http://localhost:3000/
 
+### Lando
+
+1. Uninstall Docker
+2. Install Lando DMG from [[https://github.com/lando/lando/releases]]
```

Comment:
What version of lando?

Reply:
I used the latest `v3.0.0-rrc.4` and it works well.